app/config.go

```diff
@@ -60,6 +61,10 @@ type InfraConfig struct {
 	STUNServers []string
 }
 
+type OperationsConfig struct {
+	CreateHostDisabled bool
```

Comment:
Use `CreateHostEnabled` instead.

I think I've seen a recommendation not to use negative (disabled) in config properties, but I couldn't find any official guidance. The idea is that using "Disabled" makes it confusing: for example, the docs could say "When `CreateHostDisabled` is disabled Host Creation is enabled".

Reply:
I see this relevant when the config parameter always needs to be set, in this case it could be omitted which makes a better default IMO vs having to set `CreateHostEnabled = true` every time. Using the omitting way docs should only point out that if you want to disable host creation add `CreateHostDisabled = true`.